components/navigable-container/index.js

```diff
@@ -160,9 +164,9 @@ export class NavigableMenu extends Component {
 
 export class TabbableContainer extends Component {
 	render() {
-		const eventToOffset = ( evt ) => {
-			const { keyCode, shiftKey } = evt;
-			if ( TAB === keyCode ) {
+		const eventToOffset = ( evt, container ) => {
+			const { keyCode, shiftKey, target } = evt;
+			if ( TAB === keyCode && target.parentElement === container ) {
```

Comment:
The tabbable container behavior triggers only if we're currently focusing one of its children.